clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/models/output/SCVTraitSet.scala

```diff
@@ -28,23 +30,37 @@ object SCVTraitSet {
   implicit val encoder: Encoder[SCVTraitSet] = deriveEncoder(renaming.snakeCase, None)
 
   /** Extract a TraitSet model from a raw TraitSet payload which was nested under a ClinicalAssertion. */
-  def fromRawAssertionSet(scv: SCV, rawSet: Msg): SCVTraitSet =
-    fromRawSet(scv.id, Some(scv.id), None, rawSet)
+  def fromRawAssertionSet(
+    scvAccessionId: String,
+    rawSet: Msg,
+    traitIds: Array[String]
+  ): SCVTraitSet =
+    fromRawSet(scvAccessionId, Some(scvAccessionId), None, rawSet, traitIds)
 
   /** Extract a TraitSet model from a raw TraitSet payload which was nested under observation data. */
-  def fromRawObservationSet(observation: SCVObservation, rawSet: Msg): SCVTraitSet =
-    fromRawSet(observation.id, None, Some(observation.id), rawSet)
+  def fromRawObservationSet(
+    observation: SCVObservation,
+    rawSet: Msg,
+    traitIds: Array[String]
```

Comment:
```suggestion
    scvTraitIds: Array[String]
```
For the sake of our future selves